scripts/monitoring/auto_scale_workers.py

```diff
@@ -2,22 +2,34 @@
 import time
 import pytz
 import requests
-
-import boto3
-
-import botocore
+import warnings
 
 from auto_stop_workers import start_worker, stop_worker
+from prometheus_api_client import PrometheusConnect
 
-utc = pytz.UTC
+warnings.filterwarnings("ignore")
 
+utc = pytz.UTC
+NUM_PROCESSED_SUBMISSIONS = "num_processed_submissions"
+NUM_SUBMISSIONS_IN_QUEUE = "num_submissions_in_queue"
+PROMETHEUS_URL = os.getenv(
+    "MONITORING_API_URL"
+)  # "https://monitoring-staging.eval.ai/prometheus/"
```

Comment:
Can you remove the comment? Also, can you set the default value of env variable to be staging endpoint? You can use `os.environ.get` to do that. See this example https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/aws_utils.py#L40